Display.cpp

```diff
@@ -0,0 +1,92 @@
+#include "Display.h"
+
+using namespace std;
+
+Display::Display(vector<LogLine>& set) : dataset(set) {}
+
+void Display::printPV(string pid, string vid) {
+    for(auto index : dataset)
+    {
+        if((index.getProcessId() == pid) && (index.getThreadId() == vid))
+        {
+            print(index);//write final print function
+        }
+    }
+}
+void Display::printTimeRange(string start, string end) {
+    for(auto index : dataset)
+    {
+        doParsing(start);
+        map<string,int> uTimemap;
+        uTimemap = index.getTime();
+        if((uTimemap.contains("Hour")) && (uTimemap.contains("Minute")) && (uTimemap.contains("Second")) && (uTimemap.contains("Millisecond")))
+        {
+            if((uTimemap["Hour"]>=hour) && (uTimemap["Minute"]>=minute) && (uTimemap["Second"]>=second) && (uTimemap["Millisecond"]>=millisecond))
+            {
+                doParsing(end);
+            }
+            if((uTimemap["Hour"]<=hour) && (uTimemap["Minute"]<=minute) && (uTimemap["Second"]<=second) && (uTimemap["Millisecond"]<=millisecond))
```

Comment:
This if condition has to be inside previous if { }

Reply:
done